code/frameworks/angular/src/client/angular-beta/ComputesTemplateFromComponent.test.ts

```diff
@@ -324,7 +324,9 @@ describe('angular source decorator', () => {
           type: {
             name: 'enum',
             required: true,
-            summary: 'ButtonAccent',
+            value: [],
+            // Not allowed on the type
+            // summary: 'ButtonAccent',
```

Comment:
What to do with this?

Reply:
I'm not quite sure. The actual code reads that field (`summary`) also but I'm not really sure how it works, given that field doesn't exist on the `argType`. How would it get there I wonder?

https://github.com/storybookjs/storybook/blob/506702abbd74e69c4e25e4728dc031b751e1d047/code/frameworks/angular/src/client/angular-beta/ComputesTemplateFromComponent.ts#L76-L79

The `extractType` function does not appear to ever add a `summary` field:

https://github.com/storybookjs/storybook/blob/506702abbd74e69c4e25e4728dc031b751e1d047/code/frameworks/angular/src/client/docs/compodoc.ts#L140-L159

Reply:
Maybe it't something users specify manually, when they write argTypes themselves?

I've never seen this before.

Reply:
Do you think I should

a) delete all the summary code
b) delete it from the test, but leave it in the formatting code, just in case
c) assume it's something users might specify and extend the type for this use case? (and restore the test?)

I'm inclined to do (a) and see if we hear anything, TBH. Perhaps @yngvebn or @ThibaudAV could weigh in here?

Reply:
Yeah, I'm also inclined to remove the code